Fix node services title and cards

alexandercyu opened the pull request:
Fixes #141 
Fixes #142 
- Fix node services title font and spacing
- Fix node services cards alignment

mishaschwartz reviewed (changes requested):
This changes the spacing of these cards when the screen width is `> sm`. Specifically, buttons are too close together: 

![image](https://github.com/DACCS-Climate/marble-website/assets/4380924/a59bd73c-31f3-4317-89e3-4347df2daf81)

alexandercyu commented on static/css/marble.css:
```diff
@@ -1096,6 +1096,50 @@ h6{
+        padding-bottom: 15px;
+    }
+
+    .margin-node-card-title{
```
The widths of the cards are different at different screen sizes.  sm is 345px, lg is 322px.  The paddings are different at different screen sizes as well.

alexandercyu replied:
Moved out of media queries.

alexandercyu commented:
> This changes the spacing of these cards when the screen width is `> sm`. Specifically, buttons are too close together:
> 
> ![image](https://private-user-images.githubusercontent.com/4380924/334933412-a59bd73c-31f3-4317-89e3-4347df2daf81.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTcwMTM3MjgsIm5iZiI6MTcxNzAxMzQyOCwicGF0aCI6Ii80MzgwOTI0LzMzNDkzMzQxMi1hNTliZDczYy0zMWYzLTQzMTctODllMy00MzQ3ZGYyZGFmODEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjlUMjAxMDI4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YWNhOThjMjE5MTI4NTVjNzlmODJmMjQwODI4M2E4Yjk4ZjAxOTA4MjhjN2MzOTllZDhiN2RjZTMyNjM3MzJjYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.JFsBByuxSnJvp9hLNFYpYAvxUF4EWxodhemD-4qhDhI)

Fixed the space between buttons for larger screen sizes.

ShrutiK100 reviewed (changes requested):
On laptop browser, the services are divided in to 2 columns. where as the need to be presented in 3X3. Able to see that grip on desktop but since most of our user user a laptop to browse the site we should;d focus on creating 3X3 grid presentation for the service on Laptop browser.
<img width="1388" alt="Screenshot 2024-05-30 at 10 46 52 AM" src="https://github.com/DACCS-Climate/marble-website/assets/161752175/ed6369f3-6f69-42d0-8399-4827760dde5e">

Please check the gap between the Service heading, body and buttons, needs to be 10px gap on browser site.
<img width="316" alt="Screenshot 2024-05-30 at 10 53 18 AM" src="https://github.com/DACCS-Climate/marble-website/assets/161752175/665da89f-c608-4324-940d-75ec893f9321">

Mobile version service cards looks good!

mishaschwartz commented:
@alexandercyu Here's an example of how to implement this is a dynamic grid which might get us closer to the layout that we want: https://stackoverflow.com/a/74656166/5992438

alexandercyu commented:
> On laptop browser, the services are divided in to 2 columns. Whereas, they need to be presented in 3X3 grid. Able to see that on desktop monitor(large screen) but since most of our user use a laptop to browse the site we should focus on creating 3X3 grid presentation for the services on Laptop browser as well. <img alt="Screenshot 2024-05-30 at 10 46 52 AM" width="1388" src="https://private-user-images.githubusercontent.com/161752175/335248665-ed6369f3-6f69-42d0-8399-4827760dde5e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTcwODE3NTUsIm5iZiI6MTcxNzA4MTQ1NSwicGF0aCI6Ii8xNjE3NTIxNzUvMzM1MjQ4NjY1LWVkNjM2OWYzLTZmNjktNDJkMC04Mzk5LTQ4Mjc3NjBkZGU1ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUzMFQxNTA0MTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNWI0NTEyNGExZjRlZjliM2FiODlkNjdkZWQ4MDAzYTg4MDZkNGUwNjY2OWYwMzhhMmE3YjdkM2I5MDljZTYxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.YVAfzbNoyOx19CqBatAMprxZghxX0ZZdeaMKf1cnjgc">
> 
> Please check the gap between the Service heading, body and buttons, needs to be 10px gap on browser site. <img alt="Screenshot 2024-05-30 at 10 53 18 AM" width="316" src="https://private-user-images.githubusercontent.com/161752175/335250339-665da89f-c608-4324-940d-75ec893f9321.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTcwODE3NTUsIm5iZiI6MTcxNzA4MTQ1NSwicGF0aCI6Ii8xNjE3NTIxNzUvMzM1MjUwMzM5LTY2NWRhODlmLWM2MDgtNDMyNC05NDBkLTc1ZWM4OTNmOTMyMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUzMFQxNTA0MTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04NmE5YmU0NDdjNzI1Y2JiZjUxOTdjMDhlMTFmMTQ1YjE0YzJlNTU0NzU0NzkxMjI0MDFmNTJiZDAzYTFkNzM1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.RK65f89Qb_0uUmBFEmSzOOWPejekGzW28eUBj7s_4So">
> 
> Mobile version service cards looks good!

Adjusted the spacing so it will be three per row in laptop screens.

Changed space above buttons to be 10px.

ShrutiK100 reviewed (changes requested):
Grid is fixed, looks good 
Still don't see the 10px gap between buttons
<img width="341" alt="Screenshot 2024-05-30 at 11 34 54 AM" src="https://github.com/DACCS-Climate/marble-website/assets/161752175/3c4279f0-9c82-45bd-a1c9-92b05edbe64f">

alexandercyu commented:
> Grid is fixed, looks good Still don't see the 10px gap between body and buttons <img alt="Screenshot 2024-05-30 at 11 34 54 AM" width="341" src="https://private-user-images.githubusercontent.com/161752175/335276056-3c4279f0-9c82-45bd-a1c9-92b05edbe64f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTcwODM4ODksIm5iZiI6MTcxNzA4MzU4OSwicGF0aCI6Ii8xNjE3NTIxNzUvMzM1Mjc2MDU2LTNjNDI3OWYwLTljODItNDViZC1hMWM5LTkyYjA1ZWRiZTY0Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUzMFQxNTM5NDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hNDFmZmVhZjkwNTU2Y2RhYTlhNzI1YjczNDc0YTU3YjlhZWI5NTliYmViMGQwNWVhYWNlMGI3MDc5ZGFjMzFkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9._IXUYnjHeGFtND0BW73h5SS9wnZF_Zaye-Z1Q-ldFkw">

![Screenshot 2024-05-30 at 11 46 01 AM (2)](https://github.com/DACCS-Climate/marble-website/assets/124736788/8d016b57-4e65-4e57-84fb-73f37579d990)


Should be there now.

mishaschwartz reviewed (commented):
The `p` element above the buttons has a bottom margin of 16px which is added to the 10px top margin from the buttons to increase the gap to 26px

alexandercyu commented:
> The `p` element above the buttons has a bottom margin of 16px which is added to the 10px top margin from the buttons to increase the gap to 26px

Removed the default margin from the p tag.

ShrutiK100 reviewed (approved):
Looks good!
